diff -ru fio-fio-3.15/configure fio-fio-3.15~/configure
--- fio-fio-3.15/configure	2019-07-12 09:40:36.000000000 -0700
+++ fio-fio-3.15~/configure	2019-07-17 18:03:44.358103477 -0700
@@ -149,7 +149,7 @@
 disable_native="no"
 march_set="no"
 libiscsi="no"
-prefix=/usr/local
+prefix=/usr
 
 # parse options
 for opt do
@@ -215,7 +215,6 @@
   *)
   echo "Bad option $opt"
   show_help="yes"
-  exit_val=1
   esac
 done
 
diff -ru fio-fio-3.15/Makefile fio-fio-3.15~/Makefile
--- fio-fio-3.15/Makefile	2019-07-12 09:40:36.000000000 -0700
+++ fio-fio-3.15~/Makefile	2019-07-17 18:04:28.251925871 -0700
@@ -349,7 +349,7 @@
 mandir = /usr/share/man
 sharedir = /usr/share/fio
 else
-mandir = $(prefix)/man
+mandir = $(prefix)/share/man
 sharedir = $(prefix)/share/fio
 endif
 
