diff -ru fio-fio-3.6/configure fio-fio-3.6~/configure
--- fio-fio-3.6/configure	2018-04-16 08:22:37.000000000 -0700
+++ fio-fio-3.6~/configure	2018-04-24 10:02:08.445532471 -0700
@@ -147,7 +147,7 @@
 disable_pmem="no"
 disable_native="no"
 march_set="no"
-prefix=/usr/local
+prefix=/usr
 
 # parse options
 for opt do
@@ -205,7 +205,6 @@
   *)
   echo "Bad option $opt"
   show_help="yes"
-  exit_val=1
   esac
 done
 
diff -ru fio-fio-3.6/Makefile fio-fio-3.6~/Makefile
--- fio-fio-3.6/Makefile	2018-04-16 08:22:37.000000000 -0700
+++ fio-fio-3.6~/Makefile	2018-04-24 10:02:47.495531402 -0700
@@ -316,7 +316,7 @@
 mandir = /usr/share/man
 sharedir = /usr/share/fio
 else
-mandir = $(prefix)/man
+mandir = $(prefix)/share/man
 sharedir = $(prefix)/share/fio
 endif
 
