--- fio-fio-2.10/configure.orig	2016-07-25 11:29:22.300596811 -0700
+++ fio-fio-2.10/configure	2016-07-25 11:29:46.268624838 -0700
@@ -137,7 +137,7 @@
 devdax="no"
 disable_lex=""
 disable_pmem="no"
-prefix=/usr/local
+prefix=/usr
 
 # parse options
 for opt do
@@ -186,7 +186,6 @@
   *)
   echo "Bad option $opt"
   show_help="yes"
-  exit_val=1
   esac
 done
 
--- fio-fio-2.10/Makefile.orig	2016-07-25 11:29:27.604602898 -0700
+++ fio-fio-2.10/Makefile	2016-07-25 11:29:55.708636236 -0700
@@ -292,7 +292,7 @@
 mandir = /usr/share/man
 sharedir = /usr/share/fio
 else
-mandir = $(prefix)/man
+mandir = $(prefix)/share/man
 sharedir = $(prefix)/share/fio
 endif
 
