diff -ru fio-fio-3.16/configure fio-fio-3.16~/configure
--- fio-fio-3.16/configure	2019-09-20 14:08:16.495044219 +0800
+++ fio-fio-3.16~/configure	2019-09-20 14:07:22.347602818 +0800
@@ -152,7 +152,7 @@ libiscsi="no"
 libnbd="no"
 libaio_uring="no"
 dynamic_engines="no"
-prefix=/usr/local
+prefix=/usr
 
 # parse options
 for opt do
@@ -221,7 +221,6 @@
   *)
   echo "Bad option $opt"
   show_help="yes"
-  exit_val=1
   esac
 done
 
diff -ru fio-fio-3.16/Makefile fio-fio-3.16~/Makefile
--- fio-fio-3.16/Makefile	2019-09-20 14:08:16.495044219 +0800
+++ fio-fio-3.16~/Makefile	2019-09-20 14:07:22.347602818 +0800
@@ -355,7 +355,7 @@
 mandir = /usr/share/man
 sharedir = /usr/share/fio
 else
-mandir = $(prefix)/man
+mandir = $(prefix)/share/man
 sharedir = $(prefix)/share/fio
 endif
